CLAMSCatchSummaryLoader.pyw

```diff
@@ -53,10 +54,10 @@ class CLAMSCatchSummaryLoader(QMainWindow, ui_CatchSummaryLoader.Ui_MainWindow):
         self.schemaLabel.setText('Schema: Not Connected')
 
         #  connect this GUI's button signals
-        self.connect(self.pbUpdateSurvey, SIGNAL("clicked()"), self.updateSurvey)
-        self.connect(self.pbUpdateEvent, SIGNAL("clicked()"), self.updateEvent)
-        self.connect(self.cbShip, SIGNAL("currentIndexChanged(QString)"), self.refreshSurveys)
-        self.connect(self.cbSurvey, SIGNAL("currentIndexChanged(QString)"), self.refreshEvents)
+        self.pbUpdateSurvey.clicked.connect(self.updateSurvey)
+        self.pbUpdateEvent.clicked.connect(self.updateEvent)
+        self.cbShip.clicked.connect(self.refreshSurveys)
+        self.cbSurvey.clicked.connect(self.refreshEvents)
```

Comment:
Not sure how to handle line 65: 
self.connect(startTimer, SIGNAL("timeout()"), self.startApplication)

I see in the upgrade google doc you mention using from PyQt5.QtCore import pyqtSignal and something like SerialDataReceived = pyqtSignal(str, object, object). Not sure if that's what I should be using. Was looking for examples in the existing code base, but didn't find any, if you could share any, that'd be great.

Reply:
This is the same process as the button signals you have converted. What is defined in the old SIGNAL function is the name of the signal, in this case "timeout", and it becomes the attribute of the object in the new format

OLD:
`self.connect(initTimer, SIGNAL("timeout()"), self.applicationInit)`

NEW:
`initTimer.timeout.connect(self.applicationInit)`

Reply:
okay thanks for the feedback, updated it.

Just a note, whenever I used connect I always got an error: Cannot find reference 'connect' in 'function'. So updated it to the following which I believe is correct.